specification/specs/search/search/hits/Hit.ts

```diff
@@ -1,21 +1,25 @@
 class Hit<TDocument> {
-  _explanation: Explanation;
-  fields: Dictionary<string, LazyDocument>;
-  highlight: Dictionary<string, string[]>;
-  /** @prop_serializer VerbatimInterfaceReadOnlyDictionaryKeysFormatter`2 */
-  inner_hits: Dictionary<string, InnerHitsResult>;
-  matched_queries: string[];
-  _nested: NestedIdentity;
-  _ignored: string[];
-
   _index: IndexName;
   _id: Id;
-  _shard: string;
-  _node: string;
-  _source: TDocument;
-  _seq_no: long;
-  _primary_term: long;
-  _score: double;
-  _version: long;
-  sort: Array<number | string>;
+
+  _score?: double;
+  _type?: TypeName;
+
+  _explanation?: Explanation;
+  fields?: Dictionary<string, LazyDocument>;
+  highlight?: Dictionary<string, string[]>;
+  /** @prop_serializer VerbatimInterfaceReadOnlyDictionaryKeysFormatter`2 */
```

Comment:
Should we keep this?

Reply:
For now yeah, these all indicate the .NET client does something special. Want to use these as markers to go over at some point